packages/builders/src/node/execute/node-execute.builder.spec.ts

```diff
@@ -0,0 +1,135 @@
+import {
```

Comment:
I think we should try a different way of testing it.

Basically we have several pieces of functionality in the builder:
* normalizing options, normalizing file replacements
* the conditional creation of the json file
* The creation of a webpack config

The rest is just invoking the existing builder and passing the results through.

To test it:
* we need a few good e2e tests to make sure that our assumption are correct and all integration points are correct.
* we may need a few unit tests checking the normalizing and the creation of webpack configs. If you think it's valuable to test those, I'd move those into standalone functions and test those functions.

The current tests does a lot of stubbing things out to test the happy path. Because it is stubbed out, we actually don't know if the happy path actually works (our e2e test should capture that). The current tests do not check normalize* functions.